test/core/xds/xds_metadata_test.cc

```diff
@@ -0,0 +1,276 @@
+//
+// Copyright 2022 gRPC authors.
```

Comment:
2024? looks like a new file with new contents?

Reply:
I forked this from another file and kept the original date.